src/allreduce_base.h

```diff
@@ -525,6 +540,12 @@ class AllreduceBase : public IEngine {
   utils::TCPSocket sock_listen;
   // backdoor port
   int port = 0;
+  // enable bootstrap cache 0 false 1 true
```

Comment:
Is Rabit cache used only for bootstrap? If so, we should rename `rabit_cache` to `rabit_bootstrap_cache`.

Reply:
yes, it's only for bootstrap usage